Sources/TuistCore/Automation/Extensions/AsyncThrowingStream+XcodeBuildOutput.swift

```diff
@@ -13,7 +13,7 @@ extension AsyncThrowingStream where Element == SystemEvent<XcodeBuildOutput> {
             case let .standardOutput(output):
                 let lines = output.raw.split(separator: "\n")
                 for line in lines where !line.isEmpty {
-                    logger.info("\(line)")
+                    logger.notice("\(line)")
```

Comment:
What's the difference between info and notice?

Reply:
From what I understand, the difference is very semantic. Notice is the default and should be used for information that's relevant for the user to understand the execution. Info if for information that's less relevant but that's still shown to the user. The impact for the user is really none because we don't use the difference in semantics to change the format or anything.